modules/app/read_write.py

```diff
@@ -61,9 +59,19 @@ def removeShareableFiles(self):
         """Remove local shareable files"""
         print("Remove local shareable files")
         if self.hasShareableFiles():
-            for filename in os.listdir(self.settings.filesdir):
-                file_path = os.path.join(self.settings.filesdir, filename)
+            for filename in os.listdir(self.dir):
+                file_path = os.path.join(self.dir, filename)
 
                 if os.path.isfile(file_path):
                     os.unlink(file_path)
         return
+
+    def writeFiles(self, path: Path | str, content: str):
```

Comment:
Path | str is not allowed sub 3.10 version

Reply:
Revised here https://github.com/mhoek2/tcp-share/pull/6/commits/36768a4d3d2189ecd03e611933fe53690319a70a